R/pretty_utils.R

```diff
@@ -20,6 +20,98 @@ pretty_parse <- function(txt){
   
 }
 
+#' @importFrom stringi stri_opts_brkiter
+opts <- function (x) 
+{
+  if (identical(x, "")) {
+    stringi:::stri_opts_brkiter(type = "character")
```

Comment:
`:::` throws a note and wont be accepted by CRAN. please replace with `stri_opts_brkiter <- utils::getFromNamespace('stri_opts_brkiter','stringi')`